LICENSE.md

```diff
@@ -0,0 +1,23 @@
+<!-- LICENSEFILE/ -->
+
+<h1>License</h1>
```

Comment:
why markup and not the standard license?

Reply:
cc @digitalsadhu